### PR TITLE
Added deeplink support

### PR DESCRIPTION
**Pull request by munir-rakuten**
# Description
Added deeplink support 

## Links
MINI-3289

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: testapp/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -63,4 +64,12 @@
     <string name="tut_setting">This is the first time you launch the MiniApp Demo application on this device. You need to provide your own Host Project ID and Subscription Key from &lt;font color="#BF0000">&lt;b>Rakuten&lt;/b>&lt;/font> &lt;b>App Studio&lt;/b></string>
 
     <string name="prefix_placeholder">%1$s %2$s</string>
+
+    <string name="miniappdemo_scheme">miniappdemo</string>
+    <string name="miniappdemo_host">miniapp</string>
+    <string name="miniappdemo_pathprefix">/preview/token/</string>
+
+    <string name="perm_desc_A_PERMISSION">test permission</string>
+    <string name="perm_label_A_PERMISSION">test permission</string>
```
Comment (corycaywood) [inline testapp/src/main/res/values/strings.xml:73]: I think these aren't being used for anything, right?

**Review by corycaywood — APPROVED** [review]
Looks good 👍

File: testapp/src/main/res/values/strings.xml (hunk 1 above)
Comment (munir-rakuten) [reply testapp/src/main/res/values/strings.xml:77]: oh yes..i tested custom permission but forget to remove it.. sorry.. i will remove them

**Review by Climbatize — APPROVED** [review]
LGTM

**Review by khairul-alam-licon — APPROVED** [review]
👍

**Pull request merged**